docs/src/pages/_LiveEditor.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import usePrismTheme from '@theme/hooks/usePrismTheme';
+import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
+import Highlight, { defaultProps } from "prism-react-renderer";
+import BrowserOnly from '@docusaurus/BrowserOnly';
+import styles from './styles.module.css';
+
+const code = `function VitessceConfigEditor() {
+    const bucketDomain = "https://s3.amazonaws.com"
+    const baseUrl = bucketDomain + '/vitessce-data/0.0.31/master_release/dries';
+
+    // Instantiate a view config object.
+    const vc = new VitessceConfig("My config");
+    // Add a dataset and its files.
+    const dataset = vc
+        .addDataset("Dries")
+        .addFile(baseUrl + '/dries.cells.json', 'cells', 'cells.json')
+        .addFile(baseUrl + '/dries.cell-sets.json', 'cell-sets', 'cell-sets.json');
+    // Add components.
+    const umap = vc.addView(dataset, "scatterplot", { mapping: "UMAP" });
+    const tsne = vc.addView(dataset, "scatterplot", { mapping: "t-SNE" });
+    const cellSetsManager = vc.addView(dataset, "cellSets");
+    const cellSetSizesPlot = vc.addView(dataset, "cellSetSizes");
+    // Link the zoom levels of the two scatterplots.
+    vc.linkViews([umap, tsne], ["embeddingZoom"], [2.5]);
+    // Try un-commenting the line below!
+    //vc.linkViews([umap, tsne], ["embeddingTargetX", "embeddingTargetY"], [0, 0]);
```

Comment:
Could we use the variables that contain these values instead even if we have to export them?

Reply:
I have made a new file at `src/app/constants.js` and have added the constants to the main bundle exports